src/plugins/tune/tune_impl.cpp

```diff
@@ -113,38 +114,27 @@ void TuneImpl::play_tune_async(
 
     LogDebug() << "About to send tune: " << tune_str;
 
-    if (tune_str.size() > 230) {
-        report_tune_result(callback, Tune::Result::TUNE_TOO_LONG);
+    if (tune_str.size() > MAVLINK_MSG_PLAY_TUNE_V2_FIELD_TUNE_LEN) {
```

Comment:
Actually, the size can only be `MAVLINK_MSG_PLAY_TUNE_V2_FIELD_TUNE_LEN - 1` because we need one byte for 0 termination, right?

Reply:
Haha I think we already had that discussion last October :rofl:. See here: https://github.com/mavlink/MAVSDK/pull/879#discussion_r336982971

Reply:
Ok but MAVSDK should adhere to the MAVLink spec which says there needs to be 0 termination within the 248 chars, right?
https://mavlink.io/en/messages/common.html#PLAY_TUNE_V2

Reply:
Ok!